hnn_core/network.py

```diff
@@ -251,8 +251,58 @@ def _add_external_feeds(self):
         unique_keys = sorted(self.p_unique.keys())
         self.feedname_list += unique_keys
 
-    def _create_gid_dict(self):
-        """Creates gid dicts."""
+        self._update_gid_dict()  # add feeds -> update the gid_dict
+        self.instantiate_feeds()
+
+    def instantiate_feeds(self):
+        # each trial needs unique event time vectors
+
+        self._tmp_feed_event_times = []  # reset if called again
```

Comment:
this is code smell. Was it already there?

Reply:
Nothing was already here: all this was hidden deep in `network_builder` nested loops! I've added a docstring to mention the parameter-override `n_jobs` in `dipole.py:simulate` that makes this necessary.